Scheduler/observing_blocks_auxtel/BLOCK-296/BLOCK-296_g.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "BLOCK-296_g",
+  "id": "99d226d3-f447-449f-a872-de15db82bc23",
+  "program": "BLOCK-296",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "find_target": {
+          "el": 80,
+          "az": 0,
+          "mag_limit": 8.0,
+          "radius": 2
+        }
+      }
+    },
+    {
+      "name": "auxtel/latiss_take_sequence.py",
+      "standard": true,
+      "parameters": {
+        "program": "BLOCK-296",
+        "reason": "SITCOM-1375-g",
+        "exposure_time_sequence": [
+          30,
```

Comment:
Are you counting on the central source for the psf measurement? If so, you will saturate an 8th magnitude star in 30s. If you plan to just use the field stars instead of the central source, you should definitely open the find_target parameter search as recommended above, and maybe even search at fainter magnitudes (e.g. 10-14).